reflection-config/src/main/kotlin/org/projectnessie/buildtools/reflectionconfig/ReflectionConfigTask.kt

```diff
@@ -128,6 +132,16 @@ open class ReflectionConfigTask : DefaultTask() {
       )
   }
 
+  private fun relocatedClass(clsName: String, relocations: Map<Pattern, String>): String {
+    for (e in relocations.entries) {
+      val m = e.key.matcher(clsName)
+      if (m.matches()) {
+        return m.replaceFirst(e.value)
```

Comment:
is this replacing the key? or everything that comes after?
or the whole line?

its not immediately clear after reading the javadoc :sweat_smile: 

could it make sense to have an integration test for this extension maybe?

Reply:
Yup - on the todo list for later